### DISCUSSION TITLE
combining over_switch with over_climate

### DISCUSSION THREAD

**Discussion by coldtobi** [jmcollin78/versatile_thermostat#320] [Ideas]
My Use case: The house has underfloor heating, and every room has its own heating circutit, controlled  by a "on/off" actuator.
The actuator can be contolled with an switch entity. They cannot directly be controlled by the thermostate.

The thermostate (HmIP-WTH-2), are only for reporting the temperature and be able to set the desired setpoint, but as said, they cannot control the actuator.  (They show up as climate device though.)

So it would be nice if one could use over_climate to keep the versatile thermostate in sync with the wall thermostate, but still be able to control the actuator without the need for extra automation -- as versatile thermostate has very nice features here as well.

Thanks for considering!

**Comment by jmcollin78** [comment]
Hello @coldtobi ,

You can try something like that:
1. create a VTherm over switch for all actuators,
2. create a VTherm over climate with all preceeding VTherm as underlyings AND the thermostat you have HmIP-WTH-2 (which I suppose is accessible through an climate entity in HA.

So you will have a centralisation point (the VTherm over climate) which will delegate the target temperature to its VTherm over switch and the over_switch will turn on/off the switch until target temp is reach.

I think this could work (I already have this case in the French hacf.fr forum)
